src/appengine/handlers/cron/oss_fuzz_setup.py

```diff
@@ -521,6 +524,12 @@ def sync_cf_job(project, info, corpus_bucket, quarantine_bucket, logs_bucket,
     if selective_unpack:
       job.environment_string += 'UNPACK_ALL_FUZZ_TARGETS_AND_FILES = False\n'
 
+    if 'dataflow' in info.get('fuzzing_engines'):
+      dataflow_build_bucket_path = get_build_bucket_path(
+          project, 'dataflow', 'dataflow', template.architecture),
+      job.environment_string += (
+          'DATAFLOW_BUILD_BUCKET_PATH = %s\n' % dataflow_build_bucket_path)
```

Comment:
Please do remember to update docs for this env var here (can be done later, but fyi to not forget).

docs/production-setup/setting_up_a_fuzzing_job.md
docs/reference/job_definition.md

Reply:
Thanks, filed https://github.com/google/clusterfuzz/issues/558 to remember that.